pkg/features/kube_features.go

```diff
@@ -987,8 +960,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DownwardAPIHugePages: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in v1.29
 
-	EndpointSliceTerminatingCondition: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in v1.28
```

Comment:
EndpointSliceTerminatingCondition has a block above, too - for every gate there are 2 parts.  We need to kill both

Reply:
still unresolved